migration/identity-migration/src/main/java/io/camunda/migration/identity/AuthorizationMigrationHandler.java

```diff
@@ -25,13 +25,18 @@
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
+import java.util.concurrent.atomic.AtomicInteger;
 import java.util.stream.Collectors;
 
 public class AuthorizationMigrationHandler extends MigrationHandler<Authorization> {
   private final AuthorizationServices authorizationService;
   private final ManagementIdentityClient managementIdentityClient;
   private final ConsoleClient consoleClient;
 
+  private final AtomicInteger createdAuthorizationsCount = new AtomicInteger();
```

Comment:
❓ why did you go for AtomicInteger although we don't have a concurrent setup here?